prompting/utils/config.py

```diff
@@ -434,9 +434,8 @@ def parse_probabilities(prob_list):
         default=False,
     )
 
-    # TODO: Set organic weight setting enabled by default after Aug 1, 2024.
     parser.add_argument(
-        "--neuron.organic_set_weights_enabled",
+        "--neuron.organic_set_weights_disabled",
```

Comment:
It may make more sense to set the default to be True rather than renaming and inverting the arg

Reply:
Renamed it to be the same style with `--neuron.disable_set_weights`